xml/admin_monitoring_alerting.xml

```diff
@@ -198,6 +200,56 @@ placement:
         </step>
       </procedure>
     </sect2>
+
+    <sect2 xml:id="monitoring-custom-images">
+      <title>Deploying Using Custom or Local Images</title>
+      <para>
+        To deploy custom or local container images, the images need to be set
+        in cephadm. To do so, you will need to run the following command:
+      </para>
+      <screen>&prompt.cephuser;ceph config set mgr mgr/cephadm/&lt;option_name&gt; &lt;value&gt;</screen>
```

Comment:
```suggestion
      <screen>&prompt.cephuser;ceph config set mgr mgr/cephadm/<replaceable>option_name</replaceable> <replaceable>value</replaceable></screen>

Reply:
@p-se any particular reason this one didn't go in? Otherwise, LGTM

Reply:
Just missed it. Still checking and resolving comments. It'll be in shortly.

Reply:
Done